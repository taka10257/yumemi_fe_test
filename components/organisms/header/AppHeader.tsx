import React from 'react'
import styles from './AppHeader.module.sass'

export default function AppHeader() {
  return (
    <div className={styles.root}>
      <h1 className={styles.title}>都道府県別の総人口推移</h1>
    </div>
  )
}
