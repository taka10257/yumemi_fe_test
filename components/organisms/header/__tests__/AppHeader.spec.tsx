import React from 'react'
import { shallow } from 'enzyme'
import { expect } from '@jest/globals'
import AppHeader from '../AppHeader'

describe('components/organisms/header/AppHeader', () => {
  describe('正常系', () => {
    it('初期表示', () => {
      const header = shallow(<AppHeader />)
      expect(header).toMatchSnapshot()
    })
  })
})
