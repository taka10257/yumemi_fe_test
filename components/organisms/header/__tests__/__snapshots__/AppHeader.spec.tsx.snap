// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/organisms/header/AppHeader 正常系 初期表示 1`] = `
<div
  className="root"
>
  <h1
    className="title"
  >
    都道府県別の総人口推移
  </h1>
</div>
`;
